falcon/testing/client.py

```diff
@@ -347,6 +347,13 @@ def json(self) -> Optional[Union[dict, list, str, int, float, bool]]:
 
         return json_module.loads(self.text)
 
+    def __repr__(self):
+        content_type = ''
+        if 'Content-Type' in self.headers.keys():
```

Comment:
Not that we care too much about the performance of the testing utils, but this is a fairly wasteful test for inclusion...
IMHO it is much easier to simply do `self.headers.get('Content-Type', '')`.

Reply:
i have edited it to `self.headers.get('Content-Type', '')`.